src/App/FinancialPositions/sagas.ts

```diff
@@ -33,19 +33,17 @@ import {
 
 // Adding and withdrawing funds in the central ledger are not synchronous.
 // So we poll the account endpoint a bit until there is a change before reloading the UI.
-async function pollAccountFundsUpdate(oldAccountFunds: number, dfspName: string) {
+async function pollAccountFundsUpdate(account: Account, dfspName: string) {
   await retry(
     async () => {
       const accounts = await axios.get(`${services.ledgerService.baseUrl}/participants/${dfspName}/accounts`);
       if (accounts.status !== 200) {
         throw new Error('Account poll update failed - Request Failed');
       }
 
-      const account = accounts.data.filter(
-        (acc: { ledgerAccountType: string }) => acc.ledgerAccountType === 'SETTLEMENT',
-      )[0];
+      const updatedAccount = accounts.data.filter((acc: { id: number }) => acc.id === account.id)[0];
```

Comment:
I am doing this because its always picking up the first SETTLEMENT account if a DFSP has multiple currencies